source/simulator/assembly.cc

```diff
@@ -828,6 +831,8 @@ namespace aspect
             scratch.old_pressure);
         scratch.finite_element_values[introspection.extractors.pressure].get_function_values (old_old_solution,
             scratch.old_old_pressure);
+        scratch.finite_element_values[introspection.extractors.pressure].get_function_gradients (current_linearization_point,
+            scratch.current_pressure_gradients);
```

Comment:
Ah, ok. Yes, please open a bug report and feel free to leave the code in the patch if you want.